jupyter/requirements.txt

```diff
@@ -0,0 +1,11 @@
+numpy
+pandas
+matplotlib
+seaborn
+scikit-learn
+tensorflow
+tensorflow_probability
+keras
+tqdm
+petastorm[tf]
+
```

Comment:
remove empty line :)

Reply:
Why though
I thought there should always be empty lines